setup.cfg

```diff
@@ -8,11 +8,12 @@ testpaths =
 
 [flake8]
 max-line-length = 88
+max-complexity = 10
```

Comment:
why having both `.flake8` file and this one?

Reply:
this is a good question actually, let's remove this and see what's happen. 🗡️